distribution/test/itests/test-itests-common/src/test/java/ddf/common/test/restito/ChunkedContent.java

```diff
@@ -81,57 +73,97 @@ public static Action chunkedContent(String responseMessage, Duration messageDela
      * @return composite action that holds the headers required for a plain text response as well
```

Comment:
We should either add `headerCapture` to this JavaDoc comment or get rid of the comment altogether since this is a `private` method.